### PR TITLE
Add portsdb.adoc

### PR DESCRIPTION
**Pull request by yurivict**

### CODE REVIEW

File: 2022q4/portsdb.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+=== PortsDB: Program that imports the ports tree into an SQLite database
+
+Links: +
+link:https://github.com/yurivict/freebsd-portsdb/[PortsDB] URL: link:https://github.com/yurivict/freebsd-portsdb/[https://github.com/yurivict/freebsd-portsdb/]
+
+
```
Comment (pauamma) [inline 2022q4/portsdb.adoc:6]: Spurious line break.

File: 2022q4/portsdb.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Contact: Yuri Victorovich <yuri@FreeBSD.org>
+
+I developed the PortsDB project that imports FreeBSD ports into an SQLite
+database. The port is ports-mgmt/portsdb.
+
+The database can be fully rebuilt in around 20 minutes, after which in can
```
Comment (pauamma) [inline 2022q4/portsdb.adoc:12]: s/in can/it can/

File: 2022q4/portsdb.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+be quickly (in seconds) updated with new commits.
+The database is currently updated hourly:
+https://people.freebsd.org/~yuri/ports.sqlite
+
+PortsDB can be used to query ports using SQL, as a relational database.
+External services like Repology, FreshPorts, Portscout and similar can use
```
Comment (pauamma) [inline 2022q4/portsdb.adoc:18]: s/Portscout/Portscout,/ (serial comma)

File: 2022q4/portsdb.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+queries.
+
+Such queries aren't easy to perform by grepping the ports tree.
+Cross-DB queries are also easy to do that can combine PortsDB and
+/var/db/pkg/repo-FreeBSD.sqlite and /var/db/pkg/local.sqlite.
+
+All that needs to be done to run PortsDB is ./import.sh and then ./update.sh
+
+One particular immediate problem that PortsDB aims to solve is to fix
+inaccuracy in how FreeBSD port versions are displayed by Repology.
```
Comment (pauamma) [inline 2022q4/portsdb.adoc:32]: s/inaccuracy/inaccuracies/ (or "an inaccuracy")

File: 2022q4/portsdb.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Repology uses ports INDEX which is missing some required information.
+This leads to Repology not being able to distinguish between real
+versions and intermediate and made up versions. PortsDB should allow
```
Comment (pauamma) [inline 2022q4/portsdb.adoc:35]: s/made up/made-up/

**Review by pauamma — COMMENTED** [review]
Cosmetic changes. I can apply them while/after merging if you agree.

**Comment by yurivict** [comment]
Hi @pauamma ,

I've updated the PR. I fixed most issues that you mentioned, and also reworded some places and added more content.

Please feel free to make additional corrections whole merging.

(Particularly I didn't understand the spurious line break issue.)

**Review by pauamma — APPROVED** [review]
Will do. (By "spurious", I meant there are 2 where 1 is enough.)

**Pull request merged**